event_min_seat/views/event_event_views.xml

```diff
@@ -0,0 +1,27 @@
+<odoo>
+    <record model="ir.ui.view" id="view_event_form">
+        <field name="model">event.event</field>
+        <field name="inherit_id" ref="event.view_event_form" />
+        <field name="arch" type="xml">
+            <field name="seats_limited" position="after">
+                <span
+                    attrs="{'invisible': [('seats_limited', '=', False)], 'required': [('seats_limited', '=', True)]}"
+                >from <field name="seats_min" class="oe_inline" /></span>
```

Comment:
![image](https://github.com/OCA/event/assets/5040182/aef39c0c-1bd4-452c-9eaf-f0ada87684ab)

```suggestion
                >from <field name="seats_min" class="oe_inline" />&amp;nbsp;</span>
```

Reply:
Thanks for the suggestion. I was looking for a way to separate them, but didn't find it.